apps/backend/src/parse/parse.service.ts

```diff
@@ -112,29 +111,35 @@ export class ParseService {
    */
   async parseGarmsFile(
     contents: string,
-    fileName: string,
-    locations: MinistryLocationEntity[]
+    fileName: string
   ): Promise<{ txnFile: TransactionEntity[]; txnFileDate: string }> {
     const paymentMethods = await this.paymentMethodService.getPaymentMethods();
+    const locations = await this.locationService.findAllMinistryLocations();
```

Comment:
Moved the locations lookup within each function (this way we can return the parsed data without passing in the locations - easier for logging if files are valid)